packages/fx-core/src/component/driver/aad/utility/aadAppClient.ts

```diff
@@ -89,6 +89,33 @@ export class AadAppClient {
 
     return <AADApplication>response.data;
   }
+
```

Comment:
Should we add a TODO to make sure will not forget to remove these parts when we make sure we will not use list api?